
Aside from tons of compiler warning, I have not found out via google, why the
AdvanSys driver is marked broken.

It compiles and if it will not work with the ABP940-U I have lying around I'll
have to fix it up ...

  - Rene Rebe <rene@exactcode.de>

--- linux-2.6.11/drivers/scsi/Kconfig.vanilla	2005-05-23 09:11:06.000000000 +0200
+++ linux-2.6.11/drivers/scsi/Kconfig	2005-05-23 09:12:18.000000000 +0200
@@ -382,7 +382,7 @@
 
 config SCSI_ADVANSYS
 	tristate "AdvanSys SCSI support"
-	depends on (ISA || EISA || PCI) && SCSI && BROKEN
+	depends on (ISA || EISA || PCI) && SCSI
 	help
 	  This is a driver for all SCSI host adapters manufactured by
 	  AdvanSys. It is documented in the kernel source in
